HACKING.md

```diff
@@ -0,0 +1,8 @@
+# Release checklist #
+
+- [ ] Run `npm run docs:publish`. This will create an automatic commit
+- [ ] Run `npm version` to increment the version according to semver
+      guidelines. For example: `npm version patch`. Check `npm version
+      --help` for other options.
+- [ ] Push.
+- [ ] Run `npm publish`.
```

Comment:
Hmm.... Can we automate all of this? A script that must be used for bumping the version that regens docs in the process should ensure we never have stale docs.